web/src/app/user/duck/operations.js

```diff
@@ -32,7 +34,19 @@ const getPublicBounties = (page, pageSize, twitchUsername, state = null) => (dis
   })
 };
 
+const getPublicActiveBounty = (username) => (dispatch) => {
+  dispatch(requestPublicActiveBounty());
+  const page = 0;
+  const pageSize = 10;
```

Comment:
How would we get page 2 of PublicActiveBounties?

Reply:
Current setup is to only allow 1 active bounty. We can add more in the future but the UI will need to change as well